### DISCUSSION TITLE
Rewrite Description on Home page

### DISCUSSION THREAD

**Discussion by toby63** [mumble-voip/mumble-www#131] [General]
Some notes: 

> Mumble was the first VoIP application to establish true low latency voice communication over a decade ago. 

Is that even true?

>But low latency and gaming are not the only use cases it shines in.

You wanted to get rid of the "only" gaming affiliation, so I wouldn't note it here.

> We heard from users who record podcasts with our multi-channel audio recorder, players seeking realism with our positional audio in games, Eve Online players with huge communities of over 100 simultaneous voice participants (I bet they make good use of our extensive permission system 😄), the competitive Team Fortress 2 community making us their required voice communication platform, hobby radio transmission users, and a variety of workplaces adapting Mumble to fit their needs - be it on-head mobile devices or communicating across countries or into airplanes.

I get the intention here; it is meant to be a likeable, "flowing" text, but (sry to say it), it sounds a bit like advertising and a bit unprofessional at the same time to me.
I would simply list special features and usecases in a classical list.

Something like:
```
You can:
- record podcasts with our multi-channel audio recorder
- use realistic communication in games with our positional audio plugins
- manage large communities with our extensive permission system
```

> Administrators appreciate Mumble for being able to self host and have control over data security and privacy. Some make use of the extensive permission system for complex scenarios (for example separating two groups but leaders being able to talk to both). Some love to provide their users with additional functionality with scripts making use of server APIs, or host music bots and the like that connect to the server. Those that have an existing user database often make use of authenticators to allow authenticating with existing account login data.

I would also like a list of features better.

Like:
```
For Admins Mumble offers:
- Self-Hosting to ensure data security and privacy
- extensive permission system for complex scenarios (duplicate, already mentioned above)
- Customize your Mumble Server:
    - Use or create Scripts that interact with our Server API
    - Music Bots
    - External Authenticators    
```

And yes I know that some of these things are also mentioned below the text on the homepage, so this should be thought of as well.

> Administrators appreciate Mumble for being able to self host and have control over data security and privacy.

This part `have control over data security and privacy` is a bit misleading.
Maybe we could write something like "...to ensure data security and privacy".

**Comment by Krzmbrzl** [comment]
I agree

----

> Is that even true?

Idk for sure but I think so. It's probably more than a decade ago by now though :thinking:
